src/api/index.ts

```diff
@@ -10,12 +10,12 @@
 */
 
 // Module exports
-export * from "./Action";
-export * from "./Tasks";
-export * from "./Task";
-export * from "./RunConsole";
-export * from "./Config";
-export * from "./ActionValidator";
+export { default as Action } from "./Action";
```

Comment:
2 questions:
- Is this the same as `export * as Action from "./Action";`?
- Do users of this plugin require a certain version of typescript to be able to import modules that are `exported as`?

Reply:
1. No, I tried that and I think it ended up exporting the class in such a way that it would be accessed as "Action.default" instead of "Action".
2. I don't believe so, `export * as Action` requires TS 3.8, but tested `export { default as Action}` and it works in TS 3.7.

Reply:
Thanks for the details. I will use these as reference 😋